HHS/Head-Start-TTADP
# Add an ability to turn on/off the anniversary statistics flag for all users

## Description of change
Adds an ability to turn on/off the anniversary statistics flag for all users in the admin UI. Implemented buttons to turn on or off the anv_statistics feature flag for all users since we will be turning it off soon, but perhaps re-use again next year.


## How to test
- navigate to the admin -> feature flags page
- made sure the buttons turn the flag on or off for all users as expected

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1576


## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [n/a] UI review complete

### Production Deploy


### After merge/deploy
